grapher/stackedCharts/AbstractStackedChart.tsx

```diff
@@ -186,22 +189,16 @@ export class AbstactStackedChart
         return axis
     }
 
-    @computed private get yColumnsInSelectionOrder() {
-        // For stacked charts, we want the first selected series to be on top, so we reverse the order of the stacks.
-        const slugsInSelectionOrder = this.manager.selectedColumnSlugs?.length
-            ? this.manager.selectedColumnSlugs
-            : this.yColumnSlugs
-        return this.transformedTable.getColumns(slugsInSelectionOrder).reverse()
-    }
-
     @computed private get columnsAsSeries() {
-        return this.yColumnsInSelectionOrder.map((col) => {
-            return {
-                isProjection: col.isProjection,
-                seriesName: col.displayName,
-                rows: col.owidRows,
-            }
-        })
+        return this.yColumns
```

Comment:
Yes, I moved its definition to the `Grapher` class:

https://github.com/owid/owid-grapher/blob/27a3bde6bd901bd4ea10ded034e38ad5ef7918ad/grapher/core/Grapher.tsx#L1156-L1160

Which is then used in `AbstractStackedChart#yColumns`:

https://github.com/owid/owid-grapher/blob/81822d9802ac642d92b41ed75b2208c384da47a0/grapher/stackedCharts/AbstractStackedChart.tsx#L103-L112

...because I needed `yColumnsInSelectionOrder` in the `StackedDiscreteBar` which doesn't extend `AbstractStackedChart` (because it's technically an `AbstractTimeSeriesStackedChart`)